src/App.tsx

```diff
@@ -17,18 +80,36 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                onQueryChange={setQuery}
+                filterBy={filterBy}
+                setFilterBy={setFilterBy}
```

Comment:
don't pass function for updating state in child component, create wrapper for it, fix in all places